### PR TITLE
Removes Elephant Graveyard.

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Removes Elephant Graveyard and fluff papers. 

## Why It's Good For The Game
Quality control

## Changelog

:cl:
del: The Elephant Graveyard ruin has been taken out back
/:cl:

### CODE REVIEW

File: code/modules/ruins/lavalandruin_code/elephantgraveyard.dm
Comment (Zevotech) [inline code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:1]: Please move the stuff in this file that is used on other maps into their own respective structure/turf files, and remove anything else that isn't used. There's no reason to keep the code file of the ruin if we're removing it, even if it is a bit more work.

Comment (Erikafox) [reply code/modules/ruins/lavalandruin_code/elephantgraveyard.dm:1]: sighs. you're so right.

File: code/game/objects/structures/crates_lockers/crates/graves.dm [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		if(!opened)
+			if(S.tool_behaviour == cutting_tool)
+				to_chat(user, "<span class='notice'>You start start to dig open \the [src]  with \the [S]...</span>")
+				if (do_after(user,20, target = src))
+					opened = TRUE
+					locked = TRUE
+					dump_contents()
+					update_appearance()
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					if(lead_tomb == TRUE && first_open == TRUE)
```
Comment (Zevotech) [inline code/game/objects/structures/crates_lockers/crates/graves.dm]: I should have been more specific but you should remove this part that gives people the permanent trauma. It should only have been in the ruin so it has no purpose anymore.

File: code/game/objects/structures/crates_lockers/crates/graves.dm [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	name = "burial mound"
+	desc = "A marked patch of soil, adorned with a sandstone slab"
+	icon_state = "grave_lead"
+
+/obj/structure/closet/crate/grave/loot
+	name = "burial mound"
+	desc = "A marked patch of soil, showing signs of a burial long ago. You wouldn't disturb a grave... right?"
+	opened = FALSE
+
+/obj/structure/closet/crate/grave/loot/PopulateContents()  //GRAVEROBBING IS NOW A FEATURE
```
Comment (Zevotech) [inline code/game/objects/structures/crates_lockers/crates/graves.dm:83]: Also feel free to update this loot list with new stuff if you want. These graves are used pretty commonly on a few maps and it'd be neat to see them contain some new stuff.

**Review by Zevotech — APPROVED** [review]
That should be about it though. Thank you!

File: code/game/objects/structures/crates_lockers/crates/graves.dm (hunk 2 above)
Comment (Erikafox) [reply code/game/objects/structures/crates_lockers/crates/graves.dm:83]: will do tomorrow morning.

**Pull request merged**